awx/api/metadata.py

```diff
@@ -155,10 +155,7 @@ def get_field_info(self, field):
         ):
             field_info['type'] = 'id'
         elif (
-            isinstance(field, JSONField) or
-            isinstance(model_field, JSONField) or
-            isinstance(field, DRFJSONField) or
-            isinstance(getattr(field, 'model_field', None), JSONField) or
+            isinstance(field, (JSONField, JSONField, DRFJSONField, JSONField)) or
```

Comment:
This strikes me as incorrect.

Reply:
Could you please check the description [here](https://deepsource.io/gh/withshubh/awx/issue/PYL-R1701/description). If it seems wrong to you then I'll go ahead and revert this fix.

Reply:
`field` is not necessarily the same thing as `model_field`, is not necessarily the same thing as `getattr(field, 'model_field', None)`.

Also even if it were correct, folding it together like this without collapsing all of the uses of `JSONField` is weird.